extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -323,18 +323,41 @@ tags: {  # <amp-story-consent>
   html_format: AMP
   tag_name: "AMP-STORY-CONSENT"
   requires_extension: "amp-consent"
+  requires: "amp-story-consent extension .json script"
```

Comment:
I thought we don't need this? Did you say that amp-story-0.1.js include the amp-story-consent logic?